compiler/generator_rust.c

```diff
@@ -939,6 +945,7 @@ static void generate_dollar(struct generator * g, struct node * p) {
               "~Menv.limit = env.current.len();~N", p);
     generate(g, p->left);
     if (!g->unreachable) {
+        g->V[0] = p->name;
         /* Update string variable. */
         w(g, "~M~V0 = env.current.clone().into_owned();~N");
```

Comment:
Ok, that makes sense.
I aggree, that the plan to add documentation and throw a compiler error in this case would be the best solution for now.
What about the Java, Python and the JSX backends? String variables with literal assignments seem to be completly broken for them. Are there still active maintainers? Should we throw a compiler error when using string variables with these backends?